sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EventDirectorySteps.java

```diff
@@ -73,5 +81,20 @@ public EventDirectorySteps(WebDriverHelpers webDriverHelpers) {
           final String personUuid = EditEventSteps.person.getUuid();
           webDriverHelpers.waitUntilElementIsVisibleAndClickable(getByEventUuid(personUuid));
         });
+
+    When(
+        "I am accessing the event tab using the created event via api",
+        () -> {
+          webDriverHelpers.clickOnWebElementBySelector(NavBarPage.EVENTS_BUTTON);
+          String eventUuid = apiState.getCreatedEvent().getUuid();
+          String eventLinkPath = "/sormas-ui/#!events/data/";
+          webDriverHelpers.accessWebSite(environmentUrl + eventLinkPath + eventUuid);
+        });
+
+    When(
+        "I click on New Task from event tab",
+        () -> {
```

Comment:
Here you can replace the body with lambda expression, because the body contains only one line of code.